CONTRIBUTING.rst

```diff
@@ -1,53 +1,47 @@
 Contributing Guidelines
 =======================
 
-To learn more about the purpose of PEPs and how to go about writing a PEP, please
-start reading at PEP 1 (`pep-0001.txt <./pep-0001.txt>`_ in this repo). Note that
-PEP 0, the index PEP, is now automatically generated, and not committed to the repo.
+To learn more about the purpose of PEPs and how to go about writing one, please
+start reading at `PEP 1 <https://www.python.org/dev/peps/pep-0001/>`_.
+Also, make sure to check the `README <./README.rst>`_ for information
+on how to render the PEPs in this repository.
+Thanks again for your contributions, and we look forward to reviewing them!
+
 
 Before writing a new PEP
 ------------------------
 
-Has this idea been proposed on `python-ideas <https://mail.python.org/mailman/listinfo/python-ideas>`_
-and received general acceptance as being an idea worth pursuing? (if not then
-please start a discussion there before submitting a pull request).
-
-More details about it in `PEP 1 <https://www.python.org/dev/peps/pep-0001/#start-with-an-idea-for-python>`_.
+Prior to submitting a pull request here with your draft PEP, see `PEP 1
+<https://www.python.org/dev/peps/pep-0001/#start-with-an-idea-for-python>`_
+for some important steps to consider, including proposing and discussing it
+first in an appropriate venue, drafting a PEP and gathering feedback, and
+developing at least a prototype reference implementation of your idea.
 
-Do you have an implementation of your idea? (this is important for when you
-propose this PEP to `python-dev <https://mail.python.org/mailman/listinfo/python-dev>`_
-as code maintenance is a critical aspect of all PEP proposals prior to a
-final decision; in special circumstances an implementation can be deferred)
 
+Commit messages and PR titles
+-----------------------------
 
-Commit messages
----------------
-
-When committing to a PEP, please always include the PEP number in the subject
-title. For example, ``PEP NNN: <summary of changes>``.
+When adding or modifying a PEP, please always include the PEP number in the
+commit summary and pull request title.
+For example, ``PEP NNN: <summary of changes>``.
 
 
 Sign the CLA
 ------------
 
 Before you hit "Create pull request", please take a moment to ensure that this
 project can legally accept your contribution by verifying you have signed the
-PSF Contributor Agreement:
-
-    https://www.python.org/psf/contrib/contrib-form/
-
-If you haven't signed the CLA before, please follow the steps outlined in the
-CPython devguide to do so:
-
-    https://devguide.python.org/pullrequest/#licensing
+`PSF Contributor Agreement <https://www.python.org/psf/contrib/contrib-form/>`_.
```

Comment:
An interesting one, and a discussion maybe for counsel -- everything in this repo is dedicated to the public domain or CC0, so is the CLA actually needed here?

Reply:
IANAL, but the CLA still ensures contributors to affirm they own their contribution, affirms they release it under the appropriate license, and in much of the world the public domain doesn't exist, so CC0 is still a license. Its also more work to confirm it isn't necessary than to confirm that it is.

That said, it might be worth discussing, but it is well out of scope here, as these changes don't touch the substantive content, just the reST formatting, etc, and it would involve a lot more than just this file (e.g. changes to the bot, etc).

Reply:
Oh entirely out of scope but perhaps something to discuss (for the Documentation group, maybe?)

A

Reply:
IIRC I think there's already an issue open on this topic, at least for trivial commits (it wouldn't be true for the documentation as a whole)